doc/ref/helpintf.xml

```diff
@@ -90,25 +90,20 @@ any more. The book can be re-installed using <Ref Func="HELP_ADD_BOOK" />. <P/>
 <Section Label="The manual.six File">
 <Heading>The manual.six File</Heading>
 
-If a <F>manual.six</F> file for a help book is not in the format of the
-<F>gapmacro.tex</F> macros, explained in the document 
-<Q>The gapmacro.tex Manual Format</Q> (see the file 
-<F>gap4r5/doc/gapmacrodoc.pdf</F> from the tools archive 
-<C>etc/tools.tar.gz</C> which should be unpacked using the script 
-<F>etc/install-tools.sh</F>), the first non-empty line of 
-<F>manual.six</F> must be of the form
+The first non-empty line of <F>manual.six</F> should be of the form
```

Comment:
So, we throw away *mentioning* of `doc/gapmacrodoc.pdf` completely (cf. @fingolfin's comment in #1125). What to do with `doc/gapmacrodoc.tex` itself - does it stay where it is? Do we need to build PDF out of it during the release? I would leave tex for now, but do not build PDF.